src/AppBundle/Model/Event.php

```diff
@@ -752,6 +752,15 @@ public function addJob(Job $job): void
         $this->jobs->add($job);
     }
 
+    /**
+     * A convenience method to get the first Job (which should be the only one, for now).
+     * @return Job|false
+     */
+    public function getJob()
+    {
+        return $this->jobs->first();
```

Comment:
I wonder if we should have some deterministic ordering here to future proof against bugs with multiple jobs.

Reply:
I think there's also a place where this could be used on line 189 of EventDataController.php.

Reply:
Yeah, so I think we talked about this some; I wanted to change it so that it's a one-to-one relationship, so that there can ONLY be one job. That's a big-ish change, and I worry if there will be some point where we do want multiple jobs. So this is sort of like a interim thing until we decide on it once and for all.

Reply:
I just got tired of writing $event->getJobs()->first(), is all :)